### PR TITLE
feat(phrases): add french language

### PR DESCRIPTION
**Pull request by Olyno**
## Summary

This pull request adds the french translations to the project.

## Testing

/

### CODE REVIEW

File: packages/phrases/src/locales/fr/translation/admin-console/general.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const general = {
+  placeholder: 'Placeholder',
```
Comment (Olyno) [inline packages/phrases/src/locales/fr/translation/admin-console/general.ts:2]: This is not an oversight. In French, we don't have any equivalent words to my knowledge, so I leave the default term. I will propose a new translation if I find a better alternative.

**Review by gao-sun — APPROVED** [review]
you are the man

File: packages/phrases-ui/src/locales/fr.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    invalid_session:
+      'Session non trouvée. Veuillez revenir en arrière et vous connecter à nouveau.',
+  },
+};
+
+const en = Object.freeze({
+  translation,
+});
+
+export default en;
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/fr.ts:82]: should `export default fr;`

File: packages/phrases-ui/src/locales/fr.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    invalid_connector_request: 'Les données du connecteur ne sont pas valides',
+    unknown: 'Erreur inconnue. Veuillez réessayer plus tard.',
+    invalid_session:
+      'Session non trouvée. Veuillez revenir en arrière et vous connecter à nouveau.',
+  },
+};
+
+const en = Object.freeze({
+  translation,
+});
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/fr.ts:80]: Should guard type by `en` phrases.

```
const fr: typeof en = Object.freeze({
  translation,
});

export default fr;
```

File: packages/phrases/src/locales/fr/index.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import errors from './errors';
+import translation from './translation';
+
+const en = Object.freeze({
+  translation,
+  errors,
+});
+
+export default en;
```
Comment (charIeszhao) [inline packages/phrases/src/locales/fr/index.ts:9]: export default fr;

File: packages/phrases/src/locales/fr/index.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import errors from './errors';
+import translation from './translation';
+
+const fr = Object.freeze({
+  translation,
+  errors,
+});
+
+export default fr;
```
Comment (charIeszhao) [inline packages/phrases/src/locales/fr/index.ts:9]:
```suggestion
import en from './en';
import errors from './errors';
import translation from './translation';

const fr: typeof en = Object.freeze({
  translation,
  errors,
});

export default fr;
```

**Comment by gao-sun** [comment]
I helped to fix the signature - @Olyno your contribution will be preserved

**Pull request merged**